num_wheels: 4
device: eno1

modules:
  platform_driver:
    type: platform_driver
    controller: velocity_platform_controller

wheel0:
  ethercat_number: 3
  x: 0.233
  y: 0.1165
  a: 0

wheel1:
  ethercat_number: 5
  x: -0.233
  y: 0.1165
  a: 0

wheel2:
  ethercat_number: 7
  x: -0.233
  y: -0.1165
  a: 0

wheel3:
  ethercat_number: 9
  x: 0.233
  y: -0.1165
  a: 0

